src/core/biblio-db.js

```diff
@@ -95,17 +57,9 @@ export const biblioDB = {
       throw new TypeError("id is required");
     }
     const db = await this.ready;
-    return new Promise((resolve, reject) => {
-      const objectStore = db.transaction([type], "readonly").objectStore(type);
-      const range = IDBKeyRange.only(id);
-      const request = objectStore.openCursor(range);
-      request.onsuccess = () => {
-        resolve(!!request.result);
-      };
-      request.onerror = () => {
-        reject(new DOMException(request.error.message, request.error.name));
-      };
-    });
+    const objectStore = db.transaction([type], "readonly").objectStore(type);
+    const range = IDBKeyRange.only(id);
+    return !!(await objectStore.openCursor(range));
```

Comment:
I know this was in the original, but it's still a bit ugly... maybe we can do? 

```suggestion
    const result = await objectStore.openCursor(range); 
    return result ? true : false;
```

Same with other one below...

Reply:
`? true : false`? It doesn't look JavaScript-y for me... Maybe `!!result`?

Reply:
!!result wfm